tests/subsys/bootloader/bl_validation_ff_key/testcase.yaml

```diff
@@ -2,6 +2,10 @@ tests:
   bootloader.bl_validation.ff_key:
     platform_allow: nrf9160dk_nrf9160 nrf52840dk_nrf52840 nrf52dk_nrf52832
       nrf5340dk_nrf5340_cpuapp
+    integration_platforms:
```

Comment:
I don't see nrf5340dk_nrf5340_cpuapp in integration but is in allow. Is it by purpose?